v2/account_claims.go

```diff
@@ -229,6 +230,7 @@ type Account struct {
 	DefaultPermissions Permissions           `json:"default_permissions,omitempty"`
 	Mappings           Mapping               `json:"mappings,omitempty"`
 	Authorization      ExternalAuthorization `json:"authorization,omitempty"`
+	TraceDest          Subject               `json:"trace_dest,omitempty"`
```

Comment:
LGTM - but I wonder if we should have a container for this option in case there are other possible trace options. If it won't change.

Reply:
At the moment there is nothing more, but I could see that we need to add things related to tracing indeed. What would you propose? Something like:
```
Trace              *MsgTrace             `json:"trace,omitempty"`
...
type MsgTrace struct {
	Destination Subject `json:"dest,omitempty"`
}
```

Reply:
Done. Please review changes, and as I mentioned previously, if you merge, could you please (or whoever is in charge) do a release so I can reference it in the NATS Server PR that I am working on? Thanks!